build/commands/lib/config.js

```diff
@@ -145,6 +158,7 @@ const Config = function () {
   this.defaultGClientFile = path.join(this.rootDir, '.gclient')
   this.gClientFile = process.env.BRAVE_GCLIENT_FILE || this.defaultGClientFile
   this.gClientVerbose = getEnvConfig(['gclient_verbose']) || false
+  this.hostOS = getHostOS()
```

Comment:
I would not store this in a field, but just call `getHostOS()` wherever `this.hostOS` is referenced. But it's a nit.